components/LandingPage/SectionTitle.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+
+interface SectionTitleProps {
+    text: string
+}
+
+const SectionTitle = ({
```

Comment:
FYI You can do `const SectionTitle:React.FC<SectionTitleProps> = ({SectionTitleProps}) => ...`